.github/workflows/README.md

```diff
@@ -271,7 +271,6 @@ Please note that jobs with matrix need to have matrix element in the comment. Ex
 
 | Workflow name | Matrix | Trigger Phrase | Cron Status |
 |:-------------:|:------:|:--------------:|:-----------:|
-| [ PostCommit BeamMetrics Publish ](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml) | N/A |`Run Beam Metrics Deployment`| [![.github/workflows/beam_PostCommit_BeamMetrics_Publish.yml](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/beam_PostCommit_BeamMetrics_Publish.yml?query=event%3Aschedule)
 | [ PostCommit Go ](https://github.com/apache/beam/actions/workflows/beam_PostCommit_Go.yml) | N/A |`Run Go PostCommit`| [![.github/workflows/beam_PostCommit_Go.yml](https://github.com/apache/beam/actions/workflows/beam_PostCommit_Go.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/beam_PostCommit_Go.yml?query=event%3Aschedule) |
```

Comment:
We need to also update all the postcommits in this file to not have trigger phrases listed. Ideally we'd list the file to modify instead.

We should also modify the part that talks about trigger phrases at the top and explain this approach.

Reply:
updated README.md replacing trigger phrase->trigger file, added instructions on how to manually trigger precommit and postcommits